keel-sql/src/main/resources/db/changelog/20200106-tie-artifacts-to-delivery-configs.yml

```diff
@@ -0,0 +1,91 @@
+databaseChangeLog:
+  - changeSet:
+      id: tie-artifacts-to-delivery-configs
+      author: emjburns
+      changes:
+        - addColumn:
+            tableName: delivery_artifact
+            columns:
+              - column:
+                  name: delivery_config_name
```

Comment:
If 2 apps/delivery configs use the same artifact, does that currently result in 2 rows in `delivery_artifact`? I'm wondering if this should have a one-to-many relationship with `delivery_config`.

Reply:
The information in `delivery_artifact` is specific to one delivery config. Two or more delivery configs could care about an artifact in the same way, but I don't think that's a super common case and if it happens, it's not a big deal to duplicate that sorting data. I kind of feel like it would be a lot more complicated if we had a one-to-many relationship, and I don't see it having any big advantages

Reply:
In that case, the `delivery_artifact_name_type_idx` index on `delivery_artifact` (name, type) needs to lose its unique constraint.

Reply:
Great call! I'll do that.

Reply:
@asher done.